src/api/middleware/cron/index.ts

```diff
@@ -0,0 +1,19 @@
+import { cronJobScheduler } from '../../cron';
+import { accountCronSchedulerJobs } from '../../cron/account';
+import { applicationCronSchedulerJobs } from '../../cron/application';
+import { Context } from '.keystone/types'; // eslint-disable-line
+
+/**
+ * cronJobs
+ * Middleware to run cron jobs
+ * @param {Context} KeystoneJS context API
+ */
+const cronJobs = (context: Context) => {
+  console.info('Running cron jobs');
+  // account cron jobs
+  cronJobScheduler(accountCronSchedulerJobs, context);
+  // application cron jobs
+  cronJobScheduler(applicationCronSchedulerJobs, context);
```

Comment:
Could we add a space before each comment so it's easier to read?

Also I think this should be moved to `helpers` or a new `cron` directory, as this is not middleware anymore